django/db/models/base.py

```diff
@@ -1482,6 +1482,16 @@ def _check_ordering(cls):
         if not cls._meta.ordering:
             return []
 
+        if cls._meta.order_with_respect_to:
```

Comment:
This will not work. `cls._meta.ordering` is defined in `django.db.models.options.Options#_prepare`, so it will fail if I will use only `order_with_respect_to`.